recipes/telingo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = 'telingo' %}
+{% set version = '2.1.2' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/potassco/{{ name }}/archive/v{{ version }}.zip
+  sha256: c9d310e000c6cebade8c1e51fab4fb5604a32a88f259fc65ea44791e1909e647
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - telingo = telingo:main
+
+requirements:
+  host:
+    - clingo
+    - python >=3.6
+    - pip
+  run:
+    - clingo >=5.5.0
+    - python >=3.6
+
+test:
+  requires:
+    - pip
+  commands:
+    - telingo --version
+    - pip check
+
+about:
+  home:  https://potassco.org/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: A solver for temporal logic programs.
+  description: |
+    Telingo is a solver for temporal programs. It leaverages clingo's input
+    language and scripting cababilities to parse and solve programs with
+    temporal formulas. As such the input of telingo is valid clingo input
+    supporting all clingo language features like for example aggregates; only
+    the way programs are grounded and solved is adjusted.
+  license_file: LICENSE.md
```

Comment:
```suggestion
```
Duplicated from above.

Reply:
It's a pity that the linter does not find this.

Reply:
I *think* (but am by no means sure) that the YAML parsing collapses the duplicates before the linter gets them.